### ISSUE TITLE
'NoneType' object is not iterable

### ISSUE THREAD

**Issue by domsj** [openvstorage/framework#805]
Saw this error in api.log on OVH

```
2016-08-10 16:19:23 13500 +0200 - ctrl-roub-01 - 85629/140463931905360 - api/middleware - 655 - ERROR - An unhandled exception occurred: 'NoneType' object is not iterable
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/django/core/handlers/base.py", line 112, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/lib/python2.7/dist-packages/rest_framework/viewsets.py", line 78, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/django/views/decorators/csrf.py", line 57, in wrapped_view
    return view_func(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/rest_framework/views.py", line 399, in dispatch
    response = self.handle_exception(exc)
  File "/usr/lib/python2.7/dist-packages/rest_framework/views.py", line 396, in dispatch
    response = handler(request, *args, **kwargs)
  File "/opt/OpenvStorage/webapps/api/backend/decorators.py", line 443, in new_function
    return_value = f(*args, **kwargs)
  File "/opt/OpenvStorage/webapps/api/backend/decorators.py", line 79, in new_function
    return f(*args, **kw)
  File "/opt/OpenvStorage/webapps/api/backend/decorators.py", line 182, in new_function
    return f(args[0], **new_kwargs)
  File "/opt/OpenvStorage/webapps/api/backend/views/messaging.py", line 95, in wait
    messages, last_message_id = thread.value
TypeError: 'NoneType' object is not iterable
```

Version: 2.7.1-fargo.2-1

**Comment by khenderick** [comment]
Fixed by #987, packaged in openvstorage-2.7.3-rev.4070.5141207

**Comment by JeffreyDevloo** [comment]
### Information
We have not seen issue on any of our environments. This issue basically meant no response from API calls, which we require for our testing.
